### ISSUE TITLE
Handle "File not found" errors on building shared bash model

### ISSUE THREAD

**Issue by de-jcup** [de-jcup/eclipse-bash-editor#262]
## Situation
When the eclipse workspace comes out of synch with the "normal" file system, it can happen that a file is 

### Details
```java
org.eclipse.core.internal.resources.ResourceException(/__ Virtual Root __/ensure-sechub-sarif-inside-local-maven.sh)[368]: java.lang.Exception: File not found: C:\Users\the_user\somewhere\ensure-sechub-sarif-inside-local-maven.sh.
    at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
    at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
    at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:835)
    at org.eclipse.core.internal.resources.File.getContents(File.java:277)
    at org.eclipse.core.internal.resources.File.getContents(File.java:268)
    at de.jcup.eclipse.commons.workspacemodel.ModelBuilderSupport.visitResource(ModelBuilderSupport.java:297)
    at de.jcup.eclipse.commons.workspacemodel.ModelBuilderSupport.doNoResourceChangedScan(ModelBuilderSupport.java:160)
    at de.jcup.eclipse.commons.workspacemodel.ModelBuilderSupport.doNoResourceChangedScan(ModelBuilderSupport.java:149)
    at de.jcup.eclipse.commons.workspacemodel.ModelBuilderSupport$1.run(ModelBuilderSupport.java:127)
    at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

```

## Wanted
- When a file is not found, the error dialog shall not appear
- The problem shall be handled internally without any user interaction

**Comment by de-jcup** [comment]
Very "funny":

The `IFile` instances was already checked if it exists or not.... by `file.exists()`, but...

reading javadoc for `exists()`:

> ..Returns whether this resource exists in the workspace.
IResource objects are lightweight handle objects used to access resources in the workspace. However, having a handle object **does not necessarily mean the workspace really has such a resource**. ...

**Comment by de-jcup** [comment]
Can be fixed, when eclipse-commons 1.2.0 has been released
